content/discrete-math/concepts/proofs/terms/proof-by-contradiction/proof-by-contradiction.md

```diff
@@ -0,0 +1,51 @@
+---
+Title: 'Proof by Contradiction'
+Description: 'Proving a statement by assuming that it is false and showing a contradiction follows.'
+Subjects:
+  - 'Computer Science'
+  - 'Discrete Math'
+Tags:
+  - 'Arithmetic'
+  - 'Logic'
+  - 'Logical'
+  - 'Math'
+CatalogContent:
+  - 'computer-science'
+  - 'paths/computer-science'
+---
+
+**Proof by Contradiction** is a technique used in mathematical proofs to demonstrate a statement by assuming that it is false and then showing that a contradiction follows on that assumption. By proving that a contradiction would occur if a statement were false, the truth of that statement is thereby demonstrated.
+
+Proof by Contradiction is also sometimes known as **indirect proof** or **reductio ad absurdum**, meaning to reduce to an absurdity.
+
+## Steps for Proof by Contradiction
+
+1. _Supposition for reductio_: Assume that the statement is false.
```

Comment:
The meaning of the word "reductio" is unclear in this context. Can you define it somewhere?

Reply:
I've renamed this step. Hopefully the new name is clearer.